app/controllers/projects_controller.rb

```diff
@@ -1,12 +1,18 @@
 class ProjectsController < ApplicationController
 
   def new
-    @project = Project.new
-    @modules = ["BE Mod 2", "BE Mod 3", "BE Mod 4", "FE Mod 2", "FE Mod 3", "FE Mod 4", "Posse"]
+    if DemoNight.current
+      @project = Project.new
+      @current = DemoNight.current.id
+      @modules = ["BE Mod 2", "BE Mod 3", "BE Mod 4", "FE Mod 2", "FE Mod 3", "FE Mod 4", "Posse"]
+    else
+      flash[:error] = "There is no active demo night, please ask Sal to make one"
+      redirect_to root_path
+    end
   end
 
   def create
-    @project = DemoNight.current.projects.new(project_params)
+    @project = Project.new(project_params)
```

Comment:
Might change this back, but works for now. Any reason we can't leave as `DemoNight.current.projects.new(project_params)`?